agent.go

```diff
@@ -156,6 +156,22 @@ var collatedTrace = false
 // if true, coredump when an internal error occurs or a fatal signal is received
 var crashOnError = false
 
+// commType is used to denote the communication channel type used.
+type commType int
+
```

Comment:
In the commit message, you wrote:
```
The runtime should not pass this information explicilty
```
Did you mean:
```
The runtime should now pass this information explicitly
```
?

Reply:
@sboeuf Ahh bad typo!! Fixed this.

Reply:
Cool!